### ISSUE TITLE
Minimizing Hearthstone in windowed mode sometimes doesn't work with Deck Tracker running

### ISSUE THREAD

**Issue by gitbittidbit** [Epix37/Hearthstone-Deck-Tracker#1358]
windows 7 64 bit.
HDT build 0.10.16

to replicate it:
put HS in windowed mode (fullscreen unchecked).
have no other windows open for other programs.
run HS and HDT, then minimize them both.
bring up HS in a normal window (not maximized)
bring up HDT in a normal window (not maximized)
minimize HDT.
now try to minimize HS, it will minimize then immediately return back to a normal window.

**Comment by culinko** [comment]
Is this still an issue with the latest version?

**Comment by FnkyTwn** [comment]
Yes. Just experienced it myself. I've had it happen for as long as i've used HDT, I finally just got around to looking it up and found this.

It's annoying when you're at work trying to get a quick game in, and you need to suddenly minimize the game, only to have HDT open it right back up for you.. over and over and over. Eventually (after 4 tries) it'll stay minimized. 

Windows 8.1 64bit
HDT build 0.14.1

**Comment by culinko** [comment]
Do you have "Pop up Hearthstone on turn start" enabled?

**Comment by FnkyTwn** [comment]
Nope. I've never enabled that option, and it's unchecked by default.
